.github/workflows/preview-smoke-test.yml

```
@@ -0,0 +1,31 @@
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Wait for Vercel preview deployment
+        uses: patrickedqvist/wait-for-vercel-preview@v1.3.1
+        id: waitForDeployment
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          max_timeout: 300
+
+      - name: Test preview health endpoint
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:5:15: Double quote to prevent globbing and word splitting

(shellcheck)